scripts/renderers.py

```diff
@@ -76,7 +76,14 @@ def filter_and_sort_toggles(self, toggles_info_flattened_dicts, toggle_type_filt
                     data_to_render.append(toggle_dict)
 
         # sort data by either annotation_name or state_name
-        sorting_key = lambda datum: datum.get("name", "")
+        def sorting_key(datum):
+            name = datum.get("name", "")
+            # This can still occur if the "name" key exists and explicitly has None as the value.
+            # In that case the abov get will return the None.
+            if name is None:
+                name = ""
+
+            return name
```

Comment:
This is the same as `datum.get("name") or ""`, right?

Reply:
Oh good point, will update, since that's less verbose.